uchicago-library/library_website
# 592 create redirect pages within loop

Fixes #592

Creates a redirect page type for loop pages that generates sidebar links.

# Testing instructions for creating a sidebar link:
1. Navigate to Loop > Groups in the wagtail admin interface
2. Create a child page of one of the pages in Groups
3. Select Loop Redirect Page from the menu of options
4. Enter the link text under title and the URL under redirect URL
5. Populate the rest of the required fields in the content tab
6. Navigate to the promote tab and check the "Show in menus" checkbox under "for site menus"
7. Publish the page
